citysync/requirements.txt

```diff
@@ -0,0 +1,25 @@
+alembic==1.3.3
+Click==7.0
+dominate==2.4.0
+Flask==1.1.1
+Flask-Bootstrap==3.3.7.1
+Flask-Login==0.4.1
+flask-marshmallow==0.10.1
+Flask-Migrate==2.5.2
+Flask-SQLAlchemy==2.4.1
+Flask-WTF==0.14.2
+itsdangerous==1.1.0
+Jinja2==2.11.0
+Mako==1.1.1
+MarkupSafe==1.1.1
+marshmallow==3.3.0
+marshmallow-sqlalchemy==0.21.0
+pkg-resources==0.0.0
```

Comment:
This is causing an error during Docker build `ERROR: Could not find a version that satisfies the requirement pkg-resources==0.0.0`

Reply:
Thatcan be removed it was generated via `pip freeze` which I guess added all self-contained dependencies. I just ran it on an EC2 instance and it works with that line removed